packages/chains/src/bsc/addresses.ts

```diff
@@ -12,6 +12,7 @@ const chainAddresses: ChainAddresses = {
   STABLE_TOKEN: BUSD,
   W_BTC_TOKEN: BTCB,
   ALGEBRA_SWAP_ROUTER: "0x327Dd3208f0bCF590A66110aCB6e5e6941A4EfA0",
+  ALGEBRA_UNI_PROXY: "0x6B3d98406779DDca311E6C43553773207b506Fa6",
```

Comment:
We need 2 sets of these:

```
 GAMMA_ALGEBRA_SWAP_ROUTER
 GAMMA_ALGEBRA_UNI_PROXY
 GAMMA_UNISWAP_V3_SWAP_ROUTER
 GAMMA_UNISWAP_V3__UNI_PROXY
```

Reply:
The first two are for QuickSwap (Algebra implementation with Gamma farms).

The other two are for Retro.Finance (UniswapV3 implementation with Gamma farms).

See: https://github.com/ionicprotocol/contracts/pull/25/files#diff-ce3f43bc3ed7bf85556bdf6bdf676be4c81449729227c5008989d7a8f824c5f0R30-R33

For the correct addresses for polygon

Reply:
do we need these for BSC?

Reply:
Not for the time being